doc/languages-frameworks/chicken.section.md

```diff
@@ -0,0 +1,42 @@
+# CHICKEN {#sec-chicken}
```

Comment:
I still feel like that can be done later, when a Guile or Racket or etc. section is written. In particular, I don't think that it's a great idea to group languages based on linguistic or PLT similarity instead of toolchain similarity; at least when I'm in this section of the docs, it's usually in order to figure out how to do something relatively specific but well-documented in generic terms, and I want to instruct a specific toolchain in a specific way.

Reply:
Thought about this again, and given that the TOC hides sub sections, it may make chicken less discoverable, so let's keep it as is for now?!